recipes/libpysal/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "3.0.7" %}
+
+package:
+  name: libpysal
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/l/libpysal/libpysal-{{ version }}.tar.gz
+  sha256: 17b9bcc769a40bd1e5f61d453cd72e60aa440521b6d164c5ac3e8bcda50c3669
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
hi @ocefpaf , is this the new way in conda to install python projects?

Reply:
Yes. @msarahan updated conda-skeleton recently but we haven't update our docs to match that yet.

Reply:
Thanks! :)